eclipse-collections/src/main/java/org/eclipse/collections/impl/set/sorted/immutable/ImmutableTreeSet.java

```diff
@@ -167,6 +167,10 @@ public boolean equals(Object obj)
         {
             return false;
         }
+        catch (NullPointerException ignored)
+        {
+            return false;
+        }
```

Comment:
Did this come up in a test? Is it covered?

Reply:
see answer above